packages/web-components/src/button/button.styles.ts

```diff
@@ -90,6 +90,7 @@ export const styles = css`
     transition-property: background, border, color;
     transition-timing-function: ${curveEasyEase};
     cursor: pointer;
+    width: inherit;
```

Comment:
If we're moving inherit here, we should likely set *any* width and height on the host and inherit here. I'd suggest moving the `min-width` to the host as well asa well as min-height.